docs/user-guide/deployment-scenarios.md

```diff
@@ -24,6 +24,8 @@ updates](station-firmware-upgrade.md) for example.
 
 ## Scalability & Availability
 
+There are multiple strategies that you can take to improve the availability of the starter kit listed below. For all these scenarios, you can further increase the availability of a concentrator by using a highly available [LNS discovery service](lns-discovery.md).
```

Comment:
it's not for all scenarios. you can certainly enable discovery for a single LNS deployment, but that would not make much sense. I would mention this is only for multi LNS deployments (where LBS can reach multiple LNS)

Reply:
Done in b205e5a1111c951ef75588799329ae3699c8b2d7